doc/source/ipynb/ufo.rst

```diff
@@ -0,0 +1,126 @@
+
+Reconstruction with UFO
+-----------------------
```

Comment:
This is neat! Can you also add one or two sentences at the beginning for explaining what UFO is, who develops it, and the system and processor requirements?